packages/infinity-sdk/package.json

```diff
@@ -39,6 +38,9 @@
     "@pancakeswap/swap-sdk-evm": "workspace:*",
     "tsup": "^6.7.0"
   },
+  "peerDependencies": {
+    "@pancakeswap/utils": "workspace:*"
+  },
```

Comment:
Should move to dev instead of peer because this is a private pkg which is not publicly viable

Reply:
if set to peer it indicates that user of this package would need to somehow install this pkg separately